docs/advanced-features/custom-document.md

```diff
@@ -82,11 +73,25 @@ class MyDocument extends Document {
 
     return initialProps
   }
+
+  render() {
+    return (
+      <Html>
+        <Head />
+        <body>
+          <Main />
+          <NextScript />
+        </body>
+      </Html>
+    )
+  }
 }
 
 export default MyDocument
 ```
 
+> **Note**: `getInitialProps` in `_document` is not called during client-side transitions, nor when a page is [statically optimized](/docs/advanced-features/automatic-static-optimization.md).
```

Comment:
```suggestion
> **Note**: `getInitialProps` in `_document` is not called during client-side transitions.
```
`getInitialProps` in `_document` is called when prerendering during build for auto static pages.